### PR TITLE
Change failure policy of eventing webhooks to Ignore

### PR DESCRIPTION
**Pull request by devguyio**
## Proposed Changes

* Change failure policy of eventing webhooks to Ignore

### CODE REVIEW

File: cmd/eventing-operator/kodata/knative-eventing/1-eventing.yaml [hunk 1]
```diff
@@ -2768,7 +2768,7 @@ webhooks:
     service:
       name: eventing-webhook
       namespace: knative-eventing
-  failurePolicy: Fail
+  failurePolicy: Ignore
```
Comment (matzew) [inline cmd/eventing-operator/kodata/knative-eventing/1-eventing.yaml:2771]: In the past we have ONLY dones this for the sinkbinding 


please let's just use sinkbinding, for now

**Comment by matzew** [comment]
This is related to https://issues.redhat.com/browse/SRVKE-308

which was a blocker before....

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: cmd/eventing-operator/kodata/knative-eventing/1-eventing.yaml (hunk 1 above)
Comment (devguyio) [reply cmd/eventing-operator/kodata/knative-eventing/1-eventing.yaml:2771]: Done

**Pull request merged**